text/chapter13.md

```diff
@@ -81,22 +70,26 @@ To provide error messages along with our failed test cases, `purescript-quickche
 quickCheck \xs ys ->
   let
     result = merge (sort xs) (sort ys)
+    expected = sort $ xs <> ys
   in
-    xs `isSubarrayOf` result <?> show xs <> " not a subarray of " <> show result
+    eq result expected <?> "Result:\n" <> show result <> "\nnot equal to expected:\n" <> show expected
 ```
 
 This time, if we modify the code to introduce a bug, we see our improved error message after the first failed test case:
 
 ```text
-Error: Test 6 failed:
-[79168] not a subarray of [-752832,686016]
+Error: Test 1 (seed 534161891) failed:
+Result:
+[-822215,-196136,-116841,618343,887447,-888285]
+not equal to expected:
+[-888285,-822215,-196136,-116841,618343,887447]
 ```
 
-Notice how the input `xs` and `ys` were generated as a arrays of randomly-selected integers.
+Notice how the input `xs` and `ys` were generated as arrays of randomly-selected integers.
 
  ## Exercises
 
- 1. (Easy) Write a property which asserts that merging an array with the empty array does not modify the original array.
+ 1. (Easy) Write a property which asserts that merging an array with the empty array does not modify the original array. This property is redundant.
  1. (Easy) Add an appropriate error message to the remaining property for `merge`.
```

Comment:
This exercise probably ought to be removed now, I think?

Reply:
I think it's important to give the readers at least one exercise to tackle to help them absorb the material, and I think the two exercise go together nicely.

Ideally we could find some better replacement exercises. I'll think about this some more.